platform/viewer/cypress/support/commands.js

```diff
@@ -118,13 +130,24 @@ Cypress.Commands.add('waitDicomImage', (timeout = 10000) => {
 });
 
 
-//Command to reset the viewport changes throught the cornerstone method
+//Command to reset and clear all the changes made to the viewport
 Cypress.Commands.add('resetViewport', () => {
-  cy.get('@resetBtn').click()
+  cy.initCornerstoneToolsAliases();
+  cy.get('@resetBtn').click();
+  //Click on More button
+  cy.get('@moreBtn')
+    .click();
+  //Verify if overlay is displayed
+  cy.get('.tooltip-toolbar-overlay')
+    .should('be.visible');  
```

Comment:
Hmm.. If it's not, we could toggle it again?

Reply:
Yep, we can! I added a conditional test for that.